terms/BIDS_Terms/Anaesthesia.jsonld

```diff
@@ -0,0 +1,9 @@
+{
+  "@context": "https://raw.githubusercontent.com/NIDM-Terms/terms/master/context/cde_context.jsonld",
+  "@type": "PersonalDataElement",
+  "description": "OPTIONAL Details of anaesthesia used, if any.",
+  "responseOptions": [
```

Comment:
`responseOptions` is not an array of objects. so it should be just `responseOptions: {}` if any.

Reply:
@sanuann I used the online tool [here](https://nidm-terms.github.io/addNewTerm.html?) to submit this PR. The jsonld that was created and resultant empty response options are entirely out of my hands. Should I remove them? or should the submission of terms through the online tool referenced earlier instead pop any empty or null entries?

Let me know if you would like me to submit changes or if I should cancel this PR and resubmit the terms using the online tool once a fix/change has been made.

Reply:
@nqueder Can we merge this pull request?